Source/Core/Core/Debugger/PPCDebugInterface.cpp

```diff
@@ -19,26 +19,22 @@
 
 std::string PPCDebugInterface::Disassemble(unsigned int address)
 {
-	// Memory::ReadUnchecked_U32 seemed to crash on shutdown
+	// PowerPC::HostRead_U32 seemed to crash on shutdown
 	if (PowerPC::GetState() == PowerPC::CPU_POWERDOWN)
 		return "";
 
-	if (Core::GetState() != Core::CORE_UNINITIALIZED)
+	if (Core::GetState() == Core::CORE_PAUSE)
```

Comment:
The problem is the contents of the code window depends on processor state (whether address translation is turned on, MMU mappings, the contents of memory, etc.).  This state can't be safely read unless emulation is paused; otherwise, you end up with race conditions.  These races became much more evident with this PR because address translation gets turned on and off frequently, leading to PanicAlerts and inconsistent results.